buildenv/jenkins/testJobTemplate

```diff
@@ -41,6 +41,7 @@ if (!binding.hasVariable('TARGET')) TARGET = ""
 if (!binding.hasVariable('BUILD_LIST')) BUILD_LIST = ""
 if (!binding.hasVariable('SDK_RESOURCE')) SDK_RESOURCE = "upstream"
 if (!binding.hasVariable('TRIGGER_SCHEDULE')) TRIGGER_SCHEDULE = ""
+if (!binding.hasVariable('LIGHT_WEIGHT_CEHCKOUT')) LIGHT_WEIGHT_CEHCKOUT = false
```

Comment:
spelling _CEHCKOUT -> _CHECKOUT

Reply:
Thanks Shelley. I will retest and post the links.